### ISSUE TITLE
Request: Delete subscribers

### ISSUE THREAD

**Issue by suelaP** [phpList/rest-api#122]
### Short problem description
Currently, the API does not support subscriber removal.

**Comment by kniziol** [comment]
Does the goal is to create `deleteAction()` in `SubscriberController`?

**Comment by samtuke** [comment]
@kniziol Yes! The implementation is not as simple as deleting the subscriber from the `user` table. Instead the subscriber can merely be marked as blacklisted, as we should not permanently delete the preferences of a subscriber, particularly if they have opted out from future communications (we must preserve their wish).
Instead the delete procedure for each subscriber should have the same effect as this logic from phpList3: https://github.com/phpList/phplist3/blob/master/public_html/lists/admin/inc/userlib.php#L147

**Comment by TatevikGr** [comment]
Resolved: https://github.com/phpList/core/pull/343
